### PR TITLE
Add BlockRaw method

### PR DESCRIPTION
**Pull request by winder**
Add BlockRaw method to fetch the raw block bytes. Minor cleanup and extensions to the common client code to support this.

### CODE REVIEW

File: client/v2/common/common.go [hunk 1]
```diff
@@ -103,34 +103,34 @@ func mergeRawQueries(q1, q2 string) string {
 }
 
 // submitForm is a helper used for submitting (ex.) GETs and POSTs to the server
```
Comment (EvanJRichard) [inline client/v2/common/common.go:105]: Can you please fix this comment to match the function name?

Comment (EvanJRichard) [reply client/v2/common/common.go:105]: And does SubmitFormRaw need to be public like this? ("Yes" is a perfectly fine answer)

**Review by EvanJRichard — APPROVED** [review]
LGTM, noted a couple things about comments/public vs. private

File: client/v2/common/common.go (hunk 1 above)
Comment (winder) [reply client/v2/common/common.go]: Thanks, I don't think this needs to be public anymore. I was using it before adding "GetRaw"

**Pull request merged**